### ISSUE TITLE
replace stdweb with web_sys

### ISSUE THREAD

**Issue by jice-nospam** [unrust/uni-gl#4]
https://rustwasm.github.io/wasm-bindgen/examples/webgl.html

Passing buffers from rust to javascript should be much faster with this new standard API

**Comment by bwasty** [comment]
@alanpoon I see you have a branch tackling this issue. Is it working? Any insights?

**Comment by jice-nospam** [comment]
no, right now it's only an open subject. No work has been done on it yet.

**Comment by jice-nospam** [comment]
No, right now it's only a plan. No work has been done on it yet.

**Comment by Zireael07** [comment]
Any updates?

**Comment by jice-nospam** [comment]
At this point, I don't know if the switch is necessary. I though websys was the new stdweb and would replace it, but both are still relevant and tackle different issues. I lack insights to understand if the switch is worth.

**Comment by alanpoon** [comment]
> @alanpoon I see you have a branch tackling this issue. Is it working? Any insights?

I am using https://github.com/shawnscode/crayon. I discontinued using uni-gl already. The latest winit might already support web-gl. Not sure.
